ajax/libs/jquery.webticker/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "jquery.webticker",
+  "filename": "jquery.webticker.min.js",
+  "version": "3.0.0",
+  "description": "JQuery Web Ticker",
+  "keywords": [
+    "animation",
+    "ecosystem",
+    "ecosystem-jquery",
+    "horizontal",
+    "horizontal-scroll",
+    "image",
+    "image-scroll",
+    "jquery",
+    "jquery-plugin",
+    "marquee",
+    "news",
+    "news-ticker",
+    "plugin",
+    "scroll",
+    "ticker"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mazedigital/Web-Ticker.git"
+  },
+  "author": "Jonathan Mifsud",
```

Comment:
Let's modify this field as the same with upstream.
```js
  "author": {
      "name": "Jonathan Mifsud",
      "email": "jon@maze.digital",
      "url": "https://maze.digital"
  },
```
https://github.com/mazedigital/Web-Ticker/blob/master/package.json#L22-L26
Many thanks.